services/web/server/src/simcore_service_webserver/application_settings.py

```diff
@@ -182,6 +188,18 @@ def to_client_statics(self) -> Dict[str, Any]:
             exclude_none=True,
             by_alias=True,
         )
+
+        # gets names of disabled plugins
+        plugins_disabled = []
+        # NOTE: this list is limited for security reasons. An umbounded list
+        # might reveal critical info on the settings of a deploy.
+        selected_plugins = ["WEBSERVER_EXPORTER", "WEBSERVER_SCICRUNCH"]
```

Comment:
Maybe move this to a constant outside the scope of the function. Also I'd name something on the line of `PLUGIN_CANDIDATES` or `DISABLE_PLUGIN_CANDIDATES`